frontend/src/app/pages/exercises/exercise/shared/exercise-map/exercise-map.component.html

```diff
@@ -1,4 +1,5 @@
-<div #openLayersContainer class="h-100 w-100 position-relative">
+<!-- We want keyboard events to go to the OpenLayers map, not the container -> tabindex="-1" -->
```

Comment:
This line is a bit ... weird.

Reply:
```suggestion
<!-- `tabindex="-1"` because we want the keyboard events to be fired on the OpenLayers map instead of this container -->
```

Better? Else you need to be more specific than "weird"

Reply:
Ah, oh, sorry. I only now saw that the `->` is an arrow and not a HTML comment end that went badly. So it's fine then.